addons/mail/static/tests/new/composer/composer_tests.js

```diff
@@ -182,38 +180,31 @@ QUnit.test(
     "Selection is kept when changing channel and going back to original channel",
     async (assert) => {
         const pyEnv = await startServer();
-        const [firstChannelId] = pyEnv["mail.channel"].create([
-            { name: "firstChannel" },
-            { name: "secondChannel" },
+        const [channelId] = pyEnv["mail.channel"].create([
+            { name: "channel1" },
+            { name: "channel12" },
```

Comment:
12?

Reply:
also: if snake case is intended in other places, this one is lacking it

Reply:
Oops, typo